build-system/common/check-package-manager.js

```diff
@@ -332,17 +340,18 @@ function checkPythonVersion() {
       );
       console.log(
         yellow('⤷ To fix this, install the correct version from'),
-        cyan(`https://www.python.org/download/releases/${recommendedVersion}`) +
-          yellow('.')
+        cyan('https://www.python.org/downloads') + yellow('.')
       );
     }
   } else {
     console.log(
-      yellow(
-        'WARNING: ' +
-          'Could not determine the local version of python. ' +
-          'AMP development requires python 2.7.'
-      )
+      yellow('WARNING: Could not determine the local version of python.')
+    );
+    console.log(
+      yellow('⤷ To fix this, make sure'),
+      cyan(pythonExecutable),
+      yellow('is in your PATH and is version'),
```

Comment:
Optional: since `PATH` is a system constant, maybe highlight it in cyan?

Reply:
I made this change using the GitHub editor... please scrutinize.